lib/crystalball/rspec/runner.rb

```diff
@@ -17,13 +17,23 @@ def run(args, err = $stderr, out = $stdout)
 
           Crystalball.log :info, "Crystalball starts to glow..."
           prediction = build_prediction
-
+          dry_prediction(prediction)
+          
           Crystalball.log :debug, "Prediction: #{prediction.first(5).join(' ')}#{'...' if prediction.size > 5}"
           Crystalball.log :info, "Starting RSpec."
 
           super(args + prediction, err, out)
         end
 
+        def dry_prediction (prediction)
+          args = Hash[ ARGV.flat_map{|s| s.scan(/--?([^=\s]+)(?:=(\S+))?/) } ]
+          if args.key?('dry-prediction')
+            array = prediction.to_a 
+            File.write('./crystalball_spec_list.txt', array.join(","))
```

Comment:
It should write to STDOUT and the user would decide whether or not to redirect output to a file. Not sure if it would work with how the output is handled currently, in other places.